### PR TITLE
feat: support weblogic integration

### PR DESCRIPTION
**Pull request by vigohe**
### What does this PR do?

Support Weblogic integration

### CODE REVIEW

File: recipes/weblogic.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#     :name => "test"
+#   }
+# ]
+
+datadog_monitor 'weblogic' do
+  instances node['datadog']['weblogic']['instances']
+  logs node['datadog']['weblogic']['logs']
+  action :add
+  notifies :restart, 'service[datadog-agent]' if node['datadog']['agent_start']
+end
```
Comment (KSerrania) [inline recipes/weblogic.rb:42]: Is there a specific reason not to use the default integration template, as is used in many other integrations, eg. aerospike: https://github.com/DataDog/chef-datadog/blob/4c39143a77ba608d6e1265c9a06a9f04497d5cc7/recipes/aerospike.rb#L431-L438?

Using the default integration template makes it so you don't have to define parsing rules in a specific erb file, and makes the recipe future-proof (if a new option is added to the `weblogic` integration in a future Agent version, the recipe won't need to be updated to be able to use it).

**Review by KSerrania — COMMENTED** [review]
Thanks for opening this PR! I have one comment before approving.

File: recipes/weblogic.rb (hunk 1 above)
Comment (vigohe) [reply recipes/weblogic.rb:42]: Sure I will change it

Comment (vigohe) [reply recipes/weblogic.rb]: Thanks, I did change the code

**Review by KSerrania — COMMENTED** [review]
I left one last cleanup comment, otherwise LGTM.

File: templates/default/weblogic.yaml.erb
[the remark this answers is not in the document]
Comment (vigohe) [reply templates/default/weblogic.yaml.erb:1]: Thanks, I remove the file!

**Review by chouetz — APPROVED** [review]
All comments integrated

**Pull request merged**